src/Fixer/Alias/MbStrFunctionsFixer.php

```diff
@@ -42,6 +43,23 @@ final class MbStrFunctionsFixer extends AbstractFunctionReferenceFixer
         'substr_count' => ['alternativeName' => 'mb_substr_count', 'argumentCount' => [2, 3, 4]],
     ];
 
+    /**
+     * @var array<string, array>
+     */
+    private $functions;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->functions = array_filter(
+            self::$functionsMap,
+            static function (array $mapping) {
+                return \function_exists($mapping['alternativeName']) && (new \ReflectionFunction($mapping['alternativeName']))->isInternal();
```

Comment:
does it mean that we won't fix the function for end-user if he has php 7.0, but installed a polyfill ?

Reply:
Exactly, this was the purpose, wasn't it?